app/earth/index.html

```diff
@@ -46,96 +30,27 @@
   <script src="vendor/jquery.url.js"></script>
   <script src="vendor/bootstrap/js/bootstrap.js"></script>
   <script src="vendor/jquery.raty.js"></script>
-  <script src="vendor/raphael.js"></script>
-  <script src="vendor/morris.js"></script>
-  <script src="vendor/jquery.timepicker.js"></script>
-  <script src="vendor/jquery-ui-1.10.3.custom.js"></script>
-  <script src="vendor/jquery-ui.multidatespicker.js"></script>
- <script src="jquery.popupoverlay.js"></script>
-
+ 
   
       <!-- -->
   <script src="app/App.js"></script>
       
       
     <!-- Models  -->
-     <script src="app/models/NationApplication.js"></script>
      <script src="app/models/Nation.js"></script>
-     <script src="app/models/Configuration.js"></script>
-     <script src="app/models/Community.js"></script>
      <script src="app/models/Credentials.js"></script>
-     <script src="app/models/Member.js"></script>
-     <script src="app/models/report.js"></script>
-     <script src="app/models/NationReport.js"></script>
-     <script src="app/models/request.js"></script>
-     <script src="app/models/NationReportComment.js"></script>
-     <script src="app/models/Publication.js"></script>
-    
-      <script src="app/models/Resource.js"></script>
-      <script src="app/models/Course.js"></script>
```

Comment:
It might have been already fixed was Course.js is Group.js before and reason it was showing error was due to filename of Group.js not changed to Course.js.
I might be wrong but could you please confirm that there is no Group.js file. If there is then it needs to be Course.js